AppCenterCrashes/AppCenterCrashes/MSACCrashes.mm

```diff
@@ -240,6 +240,26 @@ + (void)generateTestCrash {
   }
 }
 
++ (MSACErrorReport *_Nullable)generateLiveReport {
+  if ([MSACCrashes sharedInstance].plCrashReporter == nil) {
+    MSACLogWarning([MSACCrashes logTag], @"Generate live report should be called only after App Center start.");
```

Comment:
It can also happen if you start app center but didn't include `Crashes` in service list. May be "Cannot generate a live crash report. Crashes module is not initialized yet."?